oauth_client_integration_test.go

```diff
@@ -109,6 +109,7 @@ func TestOAuthClientsCreate(t *testing.T) {
 		oc, err := client.OAuthClients.Create(ctx, orgTest.Name, options)
 		require.NoError(t, err)
 		assert.NotEmpty(t, oc.ID)
+		assert.Empty(t, oc.Name)
```

Comment:
`assert.Nil()` 💅

Reply:
It's actually not `nil` - I thought it would be, but it comes through as `""`.